components/inputs/test/input-select.visual-diff.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../typography/typography.js';
+		</script>
+		<title>d2l-input-select visual-diff tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<style>
```

Comment:
Just a copy-paste from the CSS that BSI's Sass generates today. The only thing I removed was `width: 100%` as that's a "feature" I don't want to carry over to the Lit-based stuff.